tests/tck/features/match/SeekByTag.intVid.feature

```diff
@@ -18,9 +18,7 @@ Feature: Match seek by tag
       MATCH (v:bachelor)
       RETURN id(v) AS vid, v.age AS age
       """
-    Then the result should be, in any order, and the columns 0 should be hashed:
-      | vid          | age |
-      | 'Tim Duncan' | 42  |
+    Then a SemanticError should be raised at runtime: Not found prop `age'
```

Comment:
It's runable.

Reply:
bachelor has no  age attribute， how could it be workable

Reply:
The vertex has it. I think you could check the properties in this space and report error if don't exists.